packages/documentation-site/patternfly-docs/content/design-guidelines/components/helper-text/helper-text.md

```diff
@@ -4,24 +4,22 @@ section: components
 ---
 
 ## Usage
-Helper text is most commonly used in conjuction with form elements to provide additional hints and information for the user. It can also be used to explain why certain information is being requested, how to use a specific input or to suggest helpful hints. Helper text should always be visible and appear as a subheading, unlike [popovers](https://v4-archive.patternfly.org/v4/components/popover/design-guidelines/) which display content in a new window.
+Helper text is most commonly used in conjunction with form elements to provide additional hints and information for the user. It can also be used to explain why certain information is being requested, how to use a specific input or to suggest helpful hints. Helper text should always be visible and appear as a subheading, unlike [popovers](https://www.patternfly.org/components/popover/design-guidelines/) which display content in a new window.
```

Comment:
```suggestion
Helper text is most commonly used in conjunction with form elements to provide additional hints and information for the user. It can also be used to explain why certain information is being requested, how to use a specific input, or to suggest helpful hints. Helper text should always be visible and appear as a subheading (unlike [popovers](https://www.patternfly.org/components/popover/design-guidelines/), which display content in a new window).
```